evalml/tests/component_tests/test_varmax_regressor.py

```diff
@@ -329,3 +313,44 @@ def test_varmax_regressor_can_forecast_arbitrary_dates_past_holdout(
     varmax.fit(X_train, y_train)
 
     varmax.predict(X_test)
+
+
+@pytest.mark.parametrize("use_covariates", [True, False])
+def test_varmax_regressor_prediction_intervals(use_covariates, ts_multiseries_data):
+    X_train, X_test, y_train = ts_multiseries_data(no_features=not use_covariates)
+
+    clf = VARMAXRegressor(use_covariates=use_covariates)
+
+    clf.fit(X_train, y_train)
+    result_95 = clf.get_prediction_intervals(X_test)
+
+    series_id_targets = list(result_95.keys())
+    for series in series_id_targets:
+        series_result_95 = result_95[series]
+        conf_ints = list(series_result_95.keys())
+        data = list(series_result_95.values())
+
+        assert len(conf_ints) == 2
+        assert len(data) == 2
+        assert conf_ints[0] == "0.95_lower"
+        assert conf_ints[1] == "0.95_upper"
+
+    coverages = [0.95, 0.90, 0.85]
+    results_coverage = clf.get_prediction_intervals(X_test, None, coverages)
+    predictions = clf.predict(X_test)
+
+    series_id_targets = list(results_coverage.keys())
+    for series in series_id_targets:
+        series_results_coverage = results_coverage[series]
+        conf_ints = list(series_results_coverage.keys())
+        data = list(series_results_coverage.values())
```

Comment:
This is a lot of repeated code with the check above - can we switch this to parametrize over the passed in confidence intervals?